src/popover/popover.directive.ts

```diff
@@ -106,5 +113,6 @@ export class PopoverContainer {
 			this.onClose.emit(event);
 		}
 		this.isOpen = open;
+		this.changeDetectorRef?.markForCheck();
```

Comment:
As per above changes.

```suggestion
		this.changeDetectorRef.markForCheck();
```